src/vmm/src/builder.rs

```diff
@@ -725,7 +658,7 @@ pub fn configure_system_for_boot(
 
         linux_loader::loader::load_cmdline::<crate::vstate::memory::GuestMemoryMmap>(
             vmm.guest_memory(),
-            GuestAddress(crate::arch::x86_64::layout::CMDLINE_START),
+            crate::vstate::memory::GuestAddress(crate::arch::x86_64::layout::CMDLINE_START),
```

Comment:
mh, unintended change?

Reply:
I removed `GuestAddres` import, so need to do this. Otherwise `aarch64` version will complain about unused import.

Reply:
argh, I wish that lint was more intelligent. oh well